history/index.html

```diff
@@ -3,636 +3,750 @@
 <html>
 
 <head>
-  <meta
-    http-equiv="Content-Type"
-    content="text/html; charset=utf-8"
-  />
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   <title>Data Release</title>
 </head>
 
 <body>
   <h3 style="color: #993366;">Data Release</h3>
   <ul>
+    <li><a
```

Comment:
this li should be outside the ul and the closing li tag is missing

Reply:
please make sure all html tag has the corresponding closing tag. you can check this by folding the tags in vscode if the tags are matching.
If there's any unmatching tags, the formatter cannot compile